Marlin/Configuration_adv.h

```diff
@@ -1347,6 +1347,7 @@
 
 #if ANY(HAS_DISPLAY, DWIN_LCD_PROUI, DWIN_CREALITY_LCD_JYERSUI)
   //#define SOUND_MENU_ITEM   // Add a mute option to the LCD menu
+  //#define SOUND_ON_DEFAULT  // Buzzer/speaker default enabled state
```

Comment:
I would think this should be enabled by default now that the meaning has flipped from "DISABLE_BUZZER_DEFAULT" from the initial commit.